src/en/authors-charm-metadata.md

```diff
@@ -125,23 +125,33 @@ resources:
 
 ## Payloads
 
-You can use the `payloads` section of metadata.yaml to help the user of a charm
-better understand the purpose of payloads such as LXC, KVM and docker. This is
-especially useful in large and complex deployments.
+Payloads provide a means for the charm author to get information out of the
+services deployed by a charm. This is especially useful in large and complex
```

Comment:
someone is going to complain about the use of 'services'.  Perhaps we can avoid and simplify by just saying "... information from a deployed charm"

Reply:
OK